web/changelog.json

```diff
@@ -1,5 +1,12 @@
 [
   {
+    "version": "3.8.x",
```

Comment:
True, but with our current setup, a pull request based on master will be released as is on merge.